package.json

```diff
@@ -39,5 +41,8 @@
     "metalsmith-sass": "^1.5.0",
     "metalsmith-tagcleaner": "0.0.2",
     "nunjucks": "^3.0.1"
+  },
+  "engines": {
+    "node": "8.9.4"
```

Comment:
How does this stay in sync with .nvmrc?

Reply:
currently the consensus seems to be that you have to do this by hand. There have been several proposals to make nvm use package.json engines, but nothing seems to have happened so far:

https://github.com/creationix/nvm/issues/651
https://github.com/creationix/nvm/pull/1535

Maybe we add a comment saying 'if it changes here, make sure to look at [other file] too'

Reply:
Have to do it by hand unfortunately.

Reply:
netlify fails the first time with
```
node tasks/build.js
 /opt/build/repo/node_modules/node-sass/lib/binding.js:15
       throw new Error(errors.missingBinary());
       ^
Error: Missing binding /opt/build/repo/node_modules/node-sass/vendor/linux-x64-57/binding.node
Node Sass could not find a binding for your current environment: Linux 64-bit with Node.js 8.x
Found bindings for the following environments:
  - Linux 64-bit with Node.js 6.x
This usually happens because your environment has changed since running `npm install`.
Run `npm rebuild node-sass --force` to build the binding for your current environment.
```
so I've amended the  Netflify build command to force node-sass rebuild
` npm run build; npm rebuild node-sass --force; npm run build;`

Reply:
do we need that third command `npm run build`? Or is it ok with just the first two commands?

Reply:
we can try without the last build